### PR TITLE
Feature :: Char sheet facelift and minor changes

### PR DESCRIPTION
**Pull request by schmidtas**
## Summary
Added a small facelift to show the character sheet a bit closer to Cyberpunk Red's jumpstart PreGen sheets. There is a lot of bits to tweak yet, like a mobile layout (code is ready for mobile first but I've put the desktop as priority).
Also added some small features and some packages bits.

## Changes
* Updated README file;
* Added a editorconfig file;
* Added a new logo/favicon (because it's nice);
* Facelift for base site (header and footer);
* Character sheet & LifePath facelift (mobile and tablet WIP);
* Added options for character sheet;
* Changed blank function to good ready to print PDF (I believe that this was the idea for this feature right?);
* Very small but not optimal print style (I have to tweak this for 1 page print).

## To do
* Styles for mobile and tablet;
* Better styles for print sheet
* Form to create your own character and then get the sheet;
* Some kind of bind to a character for LifePath page (maybe?);
* Add the option to hide gear/cyberware descriptions;
* Create a option to generate also a quick sheet for referee/GM use;

Any suggestion and request is very welcome!

### CODE REVIEW

File: src/assets/sass/components/_charsheet.scss [hunk 1]
```diff
@@ -0,0 +1,252 @@
+.charsheet {
+  &-stats {
+    @extend %cut-corners-box;
+
+    &__name {
```
Comment (benabus) [inline src/assets/sass/components/_charsheet.scss]: Why the double underscores?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+[*.{js,py}]
+charset = utf-8
+
+[Makefile]
+indent_style = tab
+
+[src/**.{js,vue,scss,json}]
+indent_style = space
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (schmidtas) [reply .editorconfig]: Sure! I can bring this back to 4 spaces for sure. I'm working to add the printer friendly option and the probably the create/edit form, so I will add this change as well to the code!

File: package.json [hunk 3]
```diff
@@ -1,25 +1,26 @@
 {
-    "name": "cyberpunk_char_gen",
-    "version": "0.1.0",
-    "private": true,
-    "scripts": {
-        "serve": "vue-cli-service serve",
-        "build": "vue-cli-service build"
-    },
-    "dependencies": {
-        "bootstrap": "^4.3.1",
```
[the remark this answers is not in the document]
Comment (schmidtas) [reply package.json]: Actually I've removed the bootstrap and added all custom sass styling. This is bringing the bundle size down a bit, as bootstrap have a lot of (probably) not needed styling. Also this drops the jquery and popper requirement, which are a bit heavy for loading sometimes.

File: src/assets/sass/components/_charsheet.scss (hunk 1 above)
Comment (schmidtas) [reply src/assets/sass/components/_charsheet.scss]: So this double underscores are actually something from BEM, a styling methodology/architecture to keep class specification in a lowest value possible.
Basically is a Block - Element - Modifier, where you set your block, like a style domain, the elements inside and then some possible modifier. With that we will make the code less dependent and more flexible as well!

Example:
.character-sheet-stats (block)
.character-sheet-stats__data-info (element)
.character-sheet-stats__data-info--edit (element modifier)


Here is their site if you wanna read more about this http://getbem.com/introduction/

**Pull request merged**